application/modules/users/models/Mdl_users.php

```diff
@@ -120,9 +120,21 @@ public function validation_rules()
             'user_subscribernumber' => array(
                 'field' => 'user_subscribernumber'
             ),
+            'user_invoicing_contact' => array(
+                'field' => 'user_invoicing_contact'
+            ), 
+            'user_bank' => array(
+                'field' => 'user_bank'
```

Comment:
Should we add more validation rules, like 'xss_clean' and 'trim'. It's user inputted data and we shouldn't trust it